inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/redis/RedisDataNodeOperator.java

```diff
@@ -71,6 +77,34 @@ public DataNodeInfo getFromEntity(DataNodeEntity entity) {
     @Override
     protected void setTargetEntity(DataNodeRequest request, DataNodeEntity targetEntity) {
         RedisDataNodeRequest redisDataNodeRequest = (RedisDataNodeRequest) request;
+
+        RedisClusterMode clusterMode = RedisClusterMode.of(redisDataNodeRequest.getClusterMode());
+
+        switch (clusterMode) {
+            case STANDALONE:
+                String host = redisDataNodeRequest.getHost();
+                Preconditions.expectNotBlank(host, "Redis host cannot be empty");
+                Integer port = redisDataNodeRequest.getPort();
+                expectTrue(
+                        port != null && port > 1 && port < PORT_MAX_VALUE,
```

Comment:
One of conditions is **_port > 1_**, but the error message is **_must be greater than 0_**